react/src/types/projects.ts

```diff
@@ -12,3 +12,11 @@ export interface Project {
   streetview_instances?: any;
 }
 export class Project implements Project {}
+
+export interface ProjectRequest {
+  project: Project;
+  observable?: boolean;
+  watch_content?: boolean;
```

Comment:
what about making these not optional
```suggestion
  observable: boolean;
  watch_content: boolean;
```